src/main/twirl/views/questionnaire/start.scala.html

```diff
@@ -1,16 +1,16 @@
 <div id="contentStart" class="scannable-wrapper">
-    <h1 class="heading-xlarge">Find out if your business needs to report</h1>
-    <p>Large companies and large limited liability partnerships (LLPs) may have to report on their payment terms and practices. Use this service to tell you:</p>
+    <h1 class="heading-xlarge">Check if your business needs to report</h1>
+    <p>Large companies and large limited liability partnerships (LLPs) may have to report on their payment terms and practices.</p>
+    <p>You will be asked some questions about your business to find out:</p>
     <ul class="list list-bullet">
         <li>if your business needs to report</li>
-        <li>when your next report is due and what dates it will cover</li>
-        <li>what information you’ll have to gather before you file your report</li>
+        <li>when your next report is due and what dates it should cover</li>
+        <li>what information you need to gather before you file your report</li>
     </ul>
 
-    <p class="panel">
-        You will now be asked some questions about your business.
-        <span class="bold-small">Your answers must be for your business on its own, even if it's part of a group.
-            Every business within the group will need to do this individually.</span>
+    <p>
+        Your answers must be for your business on its own, even if it's part of a group.
+        Every business within the group will need to do this individually.
```

Comment:
The copy change is correct, but they've done usability testing which says that people missed this block

Solution: apply the previous formatting applied to this block: panel and bold